package.nls.json

```diff
@@ -108,7 +108,8 @@
     "AWS.command.refreshAwsExplorer": "Refresh Explorer",
     "AWS.command.refreshCdkExplorer": "Refresh CDK Explorer",
     "AWS.command.cdk.help": "View CDK Documentation",
-    "AWS.command.ec2.connectToInstance": "Connect to EC2 Instance...",
+    "AWS.command.ec2.openTerminal": "Open terminal in EC2 instance...",
+    "AWS.command.ec2.openRemoteConnection": "Open EC2 instance in VSCode remote...",
```

Comment:
The Remote SSH extension uses the phrase "Connect in New Window...", we should try to use similar terminology.